main.py

```
@@ -36,6 +37,20 @@ def hello():
     return '<h1>Hello Netgural!</h1>'
 
 
+@app.route('/toggle_face_recognition', methods=['POST'])
+def toggle_face_recognition():
+    user = next(iter(User.query.filter_by(email=request.args['email']).all()), None)
+    if user:
+        user.face_recognition_enabled = not user.face_recognition_enabled
+        try:
+            db.session.add(user)
+            db.session.commit()
+            return jsonify({'success switching face recognitionn to': user.face_recognition_enabled})
+        except:
```

Do not use bare except, try only to catch errors which we expect to happen during try block. In this case SQLAlchemyError. I'm thinking we should also return or log the error.